src/lumigo_tracer/lambda_tracer/lambda_reporter.py

```diff
@@ -168,36 +182,135 @@ def report_json(
     return duration
 
 
+def get_span_priority(span: Dict[Any, Any]) -> int:
+    if span.get("type") == FUNCTION_TYPE:
+        return 0
+    if is_span_has_error(span):
+        return 1
+    if span.get("type") == ENRICHMENT_TYPE:
+        return 2
```

Comment:
IMO encrichment is more important than error - there is up to one enrichment, it is super small, and contain metadata on the invocation (execution tags).
This is probably more important than one more error span...

Reply:
I agree I will take this with Or and decide :)

Reply:
Or agreed